LICENSE

```diff
@@ -1,7 +1,7 @@
 Node.js is licensed for use as follows:
 
 """
-Copyright Node.js contributors. All rights reserved.
+Copyright Joyent, Inc. and other Node.js contributors. All rights reserved.
```

Comment:
The first block is the current copyright and license block, the second is the original. While almost identical they need to be separate.